src/lib/CMakeLists.txt

```diff
@@ -71,6 +71,7 @@ add_subdirectory(sensor_calibration EXCLUDE_FROM_ALL)
 add_subdirectory(slew_rate EXCLUDE_FROM_ALL)
 add_subdirectory(systemlib EXCLUDE_FROM_ALL)
 add_subdirectory(system_identification EXCLUDE_FROM_ALL)
+add_subdirectory(signal_generator EXCLUDE_FROM_ALL)
```

Comment:
Can you keep this list sorted alphabetically?

Reply:
Ok, I fixed it